package_template.json

```diff
@@ -13,13 +13,13 @@
   },
   "dependencies": {
     "compression": "^1.7.1",
-    "polka": "^0.4.0",
+    "polka": "^0.5.0",
     "sirv": "^0.4.0"
   },
   "devDependencies": {
     "npm-run-all": "^4.1.5",
-    "sapper": "^0.24.2",
-    "svelte": "^2.0.0"
+    "sapper": "alpha",
```

Comment:
Should be `^0.26.0` now.

Reply:
gonna merge this and resolve in master because i'm getting confused by the build script